training_docs/06_jupyter_notebook_workflow.md

```diff
@@ -316,6 +317,8 @@ To reload these variables at any point (e.g., if you update `catalog.yml`), use
 
 ![reload kedro line magic graphic](./images/jupyter_notebook_loading_context.png)
 
+Note that if you want to pass an argument to `reload_kedro` line magic function, you should call like a normal Python function (e.g `reload_kedro(extra_params=extra_params)` rather than using `%reload_kedro` in a notebook cell (e.g. `%reload_kedro(extra_params=extra_params)` wouldn't work).
```

Comment:
```suggestion
Note that if you want to pass an argument to `reload_kedro` line magic function, you should call it like a normal Python function (e.g `reload_kedro(extra_params=extra_params)` rather than using `%reload_kedro` in a notebook cell (e.g. `%reload_kedro(extra_params=extra_params)` wouldn't work).
```